v2.0/stop-a-node.md

```diff
@@ -14,7 +14,14 @@ For information about permanently removing nodes to downsize a cluster or react
 
 ### How It Works
 
-When you stop a node, CockroachDB lets the node finish in-flight requests and transfers all **range leases** off the node before shutting it down. If the node then stays offline for a certain amount of time (5 minutes by default), the cluster considers the node dead and starts to transfer its **range replicas** to other nodes as well.
+When you stop a node, it performs the following steps:
+
+- Finishes in-flight requests. Note that this is a best effort that times out at the `server.shutdown.query_wait` [cluster setting](cluster-settings.html).
+- Transfers all *range leases* and Raft leadership to other nodes.
+- Gossips its draining state to the cluster, so that other nodes do not try to distribute query planning to the draining node, and no leases are transferred to the draining node.  Note that this is best effort that times out at the `server.shutdown.drain_wait` [cluster setting](cluster-settings.html), so other nodes may not receive the gossip info in time.
```

Comment:
nit: Use only one space after the first period. I know you prefer 2, @rmloveland, but convention in our docs is 1.